src/status_im/subs/profile.cljs

```diff
@@ -17,6 +18,15 @@
  (fn [{:keys [customization-color]}]
    (or customization-color constants/profile-default-color)))
 
+;; A profile can only be created without accepting terms in Status v1. In Status
```

Comment:
I was confused by my own comment in the original commit, so after cherry-picking I reworded things a bit and renamed the subscription.